docs/swarm/swarm-tutorial/index.md

```diff
@@ -40,41 +40,53 @@ To run this tutorial, you need the following:
 
 ### Three networked host machines
 
-The tutorial uses three networked host machines as nodes in the swarm. These can
-be virtual machines on your PC, in a data center, or on a cloud service
-provider. This tutorial uses the following machine names:
+The tutorial uses three networked host machines as nodes in the swarm. These can be virtual machines on your PC, in a data center, or on a cloud service provider. This tutorial uses the following machine names:
 
 * manager1
 * worker1
 * worker2
 
-###  Docker Engine 1.12 or later
+>**Note:** You can follow many of the tutorial steps to test single-node swarm as well, in which case you need only one host. Multi-node commands will not work, but you can initialize a swarm, create services, and scale them.
 
-To use swarm mode, you must [install Docker Engine](../../installation/index.md)
-on each one of the host machines. Alternatively, install the latest Docker for
-Mac or Docker for Windows.
+###  Docker Engine 1.12 or newer
 
->**Note**: Docker for Mac and Docker for Windows let you use single-node
-features of swarm mode, like creating a swarm and creating a service. Multi-node
-features like joining additional nodes and scaling a service are not available.
+This tutorial requires Docker Engine 1.12 or newer on each of the host machines. Install Docker Engine and verify that the Docker Engine daemon is running on each of the machines.
 
-Verify that the Docker Engine daemon is running on each of the machines.
+You can get the latest version of Docker Engine as follows:
+
+* [install Docker Engine on Linux machines](#install-docker-engine-on-linux-machines)
+
+* [use Docker for Mac or Docker for Windows](#using-docker-for-mac-or-docker-for-windows)
+
+#### Install Docker Engine on Linux machines
+
+If you are using Linux based physical computers or cloud-provided computers as hosts, simply follow the [Linux install instructions](../../installation/index.md) for your platform. Spin up the three machines needed for this tutorial, and you are ready. You can test both single-node and multi-node swarm scenarios on Linux machines.
```

Comment:
nit: remove "needed for this tutorial". Obviously this is in the context of the tutorial.